src/controllers/user.ts

```diff
@@ -15,6 +15,14 @@ export const createUser = async (req: Request, res: Response, next: NextFunction
   try {
     const { password, username, email } = req.body;
     createUserCheck(username, password, email);
+    const existingUser = await prisma.user.findMany({
```

Comment:
For method using to check unique the prisma has check own self if already existing it will throw an Prisma.PrismaClientKnownRequestError and code=P2002